src/locales/en.js

```diff
@@ -1,8 +1,8 @@
 export const en = {
   welcome: {
     title: 'Open your new bank account',
-    description_p_1: 'To open a bank account, we will need to verify your identity.',
-    description_p_2: 'It will only take a couple of minutes.',
+    description_p_1: 'To open a bank account, we will need to verify your identity',
```

Comment:
Does the Spanish copy have the right punctuation?

Reply:
Yes you were right, pushed changes also for Spanish